src/App_test/home.test.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { render } from '@testing-library/react';
-import Home from '../components/Home';
+import Home from './mock/home';
```

Comment:
- [ ] [OPTIONAL] Well done for writing test for the `Home` component. However, it would be nice to have tests for the several components of the app not just the `Home` component
- [ ] [OPTIONAL] It would be nice to also test some pure functions in Redux code.
- [ ] [OPTIONAL] Please don't forget to write integration tests using React Testing Library.
**NB:** Kindly note that all the above are OPTIONAL simply because the previous reviewer missed them. It is very important that you try to implement them as it would be a really good learning experience for you 👍